client/my-sites/themes/theme-showcase.jsx

```diff
@@ -159,7 +159,20 @@ class ThemeShowcase extends Component {
 
 	scrollToSearchInput = () => {
 		if ( ! this.props.loggedOutComponent && this.scrollRef && this.scrollRef.current ) {
-			this.scrollRef.current.scrollIntoView();
+			// If you are a larger screen where the theme info is displayed horizontally.
+			if ( window.innerWidth > 600 ) {
```

Comment:
This seems a bit raw; does it make sense to use one of our viewport packages to achieve the same?

Reply:
I looked for a 600 px viewport but couldn't find it in the calypso library but wasn't able to find it. 
I wanted to target 600 px specifically since that is the viewport when the theme info box starts to be more horizontal. 
(Image next to the text) so the issue of the keyboard hiding the search box is not so prevelent.